smartling-api-commons/src/main/java/com/smartling/api/v2/client/DefaultClientConfiguration.java

```diff
@@ -52,4 +54,8 @@
     private HttpClientConfiguration httpClientConfiguration = new HttpClientConfiguration();
 
     private ResteasyProviderFactory resteasyProviderFactory = null;
+
+    private RestApiExceptionMapper exceptionMapper = null;
```

Comment:
I feel like we should make `DefaultRestApiExceptionMapper` the default so our exceptions are consistent across the API. What does everyone think?

Reply:
I have to look at this a bit closer but don't let it hold you up from merging once tests pass.